docs/models/components/dexprotocolsummary.md

```
@@ -0,0 +1,54 @@
+## Fields
+
+| Field                                    | Type                                     | Required                                 | Description                              |
+| ---------------------------------------- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
+| `name`                                   | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
+| `displayName`                            | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
+| `disabled`                               | *boolean*                                | :heavy_minus_sign:                       | N/A                                      |
+| `logo`                                   | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
+| `address`                                | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
+| `url`                                    | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
+| `description`                            | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
+| `audits`                                 | *string*                                 | :heavy_minus_sign:                       | N/A                                      |
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)